observability/monitoring/cluster_observability_operator/cluster-observability-operator-release-notes.adoc

```diff
@@ -16,6 +16,16 @@ The {coo-short} complements the built-in monitoring capabilities of {product-tit
 
 These release notes track the development of the {coo-full} in {product-title}.
 
+[id="cluster-observability-operator-release-notes-0-2-0"]
+== {coo-full} 0.2.0
+The following advisory is available for {coo-full} 0.2.0:
+
+* RHEA-2024:nnnn Cluster Observability Operator 0.2.0 - LINK TO BE ADDED WHEN THE ADVISORY IS LIVE
+
+[id="cluster-observability-operator-0-2-0-new-features-enhancements"]
+=== New features and enhancements
+* With this release, the {coo-full} supports installing and managing observability-related plugins for the {product-title} web console user interface (UI). (link:https://issues.redhat.com/browse/COO-58[*COO-58*])
```

Comment:
change 'observability-related plugins' to 'console-dashboards-plugin' for this release only support  'console-dashboards-plugin'

Reply:
I agree with that and keeping it generic tbh, for the very same reasons. Let's see Gabriel's thoughts.

Reply:
Yes this is generic, as the UIPlugin is an API for observability plugins not only specific to the dashboards.